recipe/meta.yaml

```diff
@@ -37,6 +37,8 @@ outputs:
     build:
       run_exports:
         - {{ pin_subpackage("libframel", max_pin="x") }}
+      ignore_run_exports:
+        - m2w64-gcc-libs  # [win]
```

Comment:
@Anthchirp, is this the recommended solution here?  I'm confused because this should really be necessary.

Reply:
I don't know if it's recommended (or even whether it works)

I do remember someone telling me that our conda-build now does more checks than were done previously, so this may be why it changed.

Reply:
so, it would work, but also needs adding for (at least) the `framel` output.